htdocs/core/class/rssparser.class.php

```diff
@@ -745,48 +752,119 @@ public function append($el, $text)
  */
 function xml2php($xml)
 {
-    $fils = 0;
-    $tab = false;
-    $array = array();
-    foreach ($xml->children() as $key => $value)
-    {
-        $child = xml2php($value);
-
-        //To deal with the attributes
-        foreach ($value->attributes() as $ak=>$av)
-        {
-            $child[$ak] = (string) $av;
-        }
-
-        //Let see if the new child is not in the array
-        if ($tab === false && in_array($key, array_keys($array)))
-        {
-            //If this element is already in the array we will create an indexed array
-            $tmp = $array[$key];
-            $array[$key] = null;
-            $array[$key][] = $tmp;
-            $array[$key][] = $child;
-            $tab = true;
-        }
-        elseif ($tab === true)
-        {
-            //Add an element in an existing array
-            $array[$key][] = $child;
-        }
-        else
-        {
-            //Add a simple element
-            $array[$key] = $child;
-        }
-
-        $fils++;
-    }
-
-
-    if ($fils == 0)
-    {
-        return (string) $xml;
-    }
-
-    return $array;
+	$fils = 0;
+	$tab = false;
+	$array = array();
+	foreach ($xml->children() as $key => $value)
+	{
+		$child = xml2php($value);
+
+		//To deal with the attributes
+		foreach ($value->attributes() as $ak=>$av)
+		{
+			$child[$ak] = (string) $av;
+		}
+
+		//Let see if the new child is not in the array
+		if ($tab === false && in_array($key, array_keys($array)))
+		{
+			//If this element is already in the array we will create an indexed array
+			$tmp = $array[$key];
+			$array[$key] = null;
+			$array[$key][] = $tmp;
+			$array[$key][] = $child;
+			$tab = true;
+		}
+		elseif ($tab === true)
+		{
+			//Add an element in an existing array
+			$array[$key][] = $child;
+		}
+		else
+		{
+			//Add a simple element
+			$array[$key] = $child;
+		}
+
+		$fils++;
+	}
+
+
+	if ($fils == 0)
+	{
+		return (string) $xml;
+	}
+
+	return $array;
+}
+
+/**
+ * Return a description/summary for one item from a ATOM feed
+ *
+ * @param	array	$item		A parsed item of a ATOM feed
+ * @param	int		$maxlength	(optional) The maximum length for the description
+ * @return	string				A summary description
+ */
+function _getAtomItemDescription(array $item, $maxlength = 500)
```

Comment:
instead of a function starting with _, i would prefer a method in class rssparser called 

private function getAtomItemDescription(...)

Same for _getAtomImageUrl()